spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.feature 'User Show', type: :feature do
+  let(:user) { User.create(name: 'Tom', photo: 'https://www.kasandbox.org/programming-images/avatars/leaf-blue.png', bio: 'He is a good programmar') }
+  let!(:post1) { Post.create(author: user, title: 'first post', text: 'first text') }
+  let!(:post2) { Post.create(author: user, title: 'second post', text: 'second text') }
+  let!(:post3) { Post.create(author: user, title: 'third post', text: '3 text') }
+  let!(:post4) { Post.create(author: user, title: '4 post', text: '4 text') }
+
+  scenario 'visiting the user Show page' do
+    visit user_path(user)
+
+    expect(page).to have_content('Tom')
+    expect(page).to have_css("img[alt='Tom']", count: 1)
+  end
+
+  scenario 'visiting the user show page, you see the number of posts the user has written..' do
+    visit user_path(user)
+
+    expect(page).to have_content('4 posts')
+  end
+
+  scenario 'visiting the user show page, you see the 3 most recent post and bio of the user has written..' do
+    visit user_path(user)
+    expect(page).to have_content('He is a good programmar')
+    expect(page).not_to have_content('fist text')
+  end
+
+  scenario 'has a link to the user index page' do
+    visit user_path(user)
+    expect(page).to have_button('See all posts')
+    click_link 'See all posts'
+    expect(current_path).to eq(user_posts_path(user))
+  end
```

Comment:
- [ ] Great effort working on the integration spec of the project milestone, but currently some of the test cases are not passing and require your attention. Checking through the test cases example shown by the `rspec spec/features` command, it shows that you have 12 examples of test cases as against the project requirements. the number of test cases required of you to carry out is around 28-29 test cases. I would kindly suggest you re-visit the project requirement and be sure to have written a test for all the cases and they are all passing. I would advise you double check the test cases you have written and have all the test cases written completely for all the files in your `spec/features`
(i.e `user_show_spec.rb`, `user_index_spec.rb`, `post_show_spec.rb`, & `post_index_spec.rb`).

Images of the test cases requested to test 👇 
![1](https://github.com/VaheStepanyan100/blog-app/assets/78634841/196dfa93-7749-4d49-8dfe-3b7e4298cd87)
![2](https://github.com/VaheStepanyan100/blog-app/assets/78634841/5b047be8-0394-4e50-a72b-36dbd8406285)


You can use the link below as a guideline.
> Note: Do not copy and paste the work in order not to be flagged for plagiarism only use it as a guideline.
https://github.com/DilsherB/ror-blog-app/pull/10/files